aspnetcore/security/authentication/scaffold-identity.md

```diff
@@ -355,6 +355,14 @@ In the *Pages/Shared/Layout.cshtml* file, make the following changes:
 
 Some Identity options are configured in *Areas/Identity/IdentityHostingStartup.cs*. For more information, see [IHostingStartup](xref:fundamentals/configuration/platform-specific-configuration).
 
+## Standalone or hosted Blazor WebAssembly apps
+
+Client-side Blazor WebAssembly apps use their own Identity UI approaches and can't use ASP.NET Core Identity scaffolding. Server-side ASP.NET Core apps of hosted Blazor solutions can follow the Razor Pages/MVC guidance in this article and are configured just like any other type of ASP.NET Core app that supports Identity.
```

Comment:
It's correct in its current form because the subject of the sentence is the server-side app of the hosted solution. The only other way to write that is to modify with the prep phrase as an opener .. 

In hosted Blazor solutions, the server-side ASP.NET Core app ...

... but I think the subject is stronger as is, and the whole paragraph already scopes the subject.